src/goto-instrument/contracts/dynamic-frames/dfcc_contract_clauses_codegen.h

```diff
@@ -8,8 +8,6 @@ Date: February 2023
 \*******************************************************************/
 
 /// \file
-/// Translates assigns and frees clauses of a function contract or
-/// loop contract into goto programs that build write sets or havoc write sets.
```

Comment:
Is this removal intentional? If so, it should be replaced by something non-empty.

Reply:
The `\file` tag is needed so that it is picked up by doxygen, but this text was moved to the class instead of the file.

I got tired of having file comments say "This file contains a class that does X, Y, Z" and have the class documentation say "this class does X, Y, Z".

Reply:
any way, the file and the class now both have the same comment again